Code/Tools/SceneAPI/SceneData/Tests/SceneManifest/SceneManifestRuleTests.cpp

```diff
@@ -502,5 +507,38 @@ namespace AZ
             scriptProcessorRuleBehavior.Deactivate();
             EXPECT_TRUE(executeByFilenameCalled);
         }
+
+        TEST_F(SceneManifest_JSON, ScriptProcessorRule_EditorPythonEventsInterface_RunsWithWhenCleared)
```

Comment:
Nit: The test name is a little confusing. Do you mean ..._RunsWithEditorPythonEventsInterfaceCleared?